tests/automation_framework/src/libs/avalon_test_wrapper.py

```diff
@@ -97,15 +97,10 @@ def pre_test_worker_env(input_file):
     response = None
     request_method = input_file.get("method")
 
-    if request_method == "WorkerRegister":
-        logger.info("No setup required for \n%s\n", request_method)
-        return 0
+    response = avalon_lib_instance.worker_lookup()
```

Comment:
Removed the pre_test_worker_env call from setup_and_build_request_worker_register